python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -6,7 +6,7 @@
 import tempfile
 import time
 import zlib
-from typing import Any, Dict, Iterator, Mapping, Optional, Sequence, cast
+from typing import Any, Dict, Iterator, Mapping, Optional, Sequence, cast, List
```

Comment:
The import could be improved into alphabetical order :)

Reply:
Done ;)